certbot-dns-digitalocean/certbot_dns_digitalocean/__init__.py

```diff
@@ -77,6 +77,15 @@
      -d example.com \\
      -d www.example.com
 
+.. code-block:: bash
+   :caption: To acquire a wildcard certificate for ``*.example.com``
+
+   certbot certonly \\
+     --dns-digitalocean \\
+     --dns-digitalocean-credentials ~/.secrets/certbot/digitalocean.ini \\
+     -d example.com \\
+     -d *.example.com
```

Comment:
Shouldn't it be enclosed in simple quotes to avoid the shell expansion on wildcard characters?

Reply:
I recently ran it successfully without any quoting but it doesn't hurt to add, so I'll add that...

Reply:
New commit pushed